sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py

```diff
@@ -510,20 +701,26 @@ def _from_generated(cls, generated):
 class HandlesPaged(PageIterator):
     """An iterable of Handles.
 
-    :ivar str marker: The continuation token of the current page of results.
-    :ivar int results_per_page: The maximum number of results retrieved per API call.
-    :ivar str continuation_token: The continuation token to retrieve the next page of results.
-    :ivar str location_mode: The location mode being used to list results. The available
-        options include "primary" and "secondary".
-    :ivar current_page: The current page of listed results.
-    :vartype current_page: list(~azure.storage.fileshare.Handle)
-
-    :param callable command: Function to retrieve the next page of items.
-    :param int results_per_page: The maximum number of share names to retrieve per
-        call.
-    :param str continuation_token: An opaque continuation token.
+    :param Callable command: Function to retrieve the next page of items.
+    :param Optional[int] results_per_page: The maximum number of share names to retrieve per call.
+    :param Optional[str] continuation_token: An opaque continuation token to retrieve the next page of results.
     """
-    def __init__(self, command, results_per_page=None, continuation_token=None):
+
+    marker: Optional[str] = None
+    """The continuation token of the current page of results."""
+    results_per_page: Optional[int] = None
+    """The maximum number of results retrieved per API call."""
+    location_mode: Optional[str] = None
+    """The location mode being used to list results.
+        The available options include "primary" and "secondary"."""
```

Comment:
Missing continuation token b/w `results_per_page` and `location_mode`?

Reply:
Gotcha! Also see below for `current_page` 😁